src/course-outline/utils/utils.js

```diff
@@ -0,0 +1,79 @@
+import { LAUNCH_CHECKLIST, BEST_PRACTICES_CHECKLIST } from '../constants';
```

Comment:
`utils/utils.js` little bit strange, wouldn't it be better to remove it?
as example `getChecklistForStatusBar.js` or something else

Reply:
Done